src/fosslight_dependency/package_manager/Npm.py

```
@@ -70,7 +70,7 @@ def install_license_checker(self):
         self._owns_local_node_dir = True
         os.environ["PATH"] = NODE_BIN + PATH_SEP + os.environ.get("PATH", "")
 
-        npm_cmd = shutil.which("npm")
+        npm_cmd = shutil.which("npm.cmd" if self.IS_WINDOWS else "npm")
         result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)